kong/metadata.csv

```diff
@@ -10,7 +16,9 @@ kong.memory.lua.shared_dict.total_bytes,gauge,,byte,,[OpenMetrics V2] The total
 kong.memory.workers.lua.vms.bytes,gauge,,byte,,[OpenMetrics V2] The allocated bytes in worker Lua VM,0,kong,,
 kong.nginx.http.current_connections,gauge,,connection,,[OpenMetrics V2] The number of HTTP connections,0,kong,,
 kong.nginx.stream.current_connections,gauge,,connection,,[OpenMetrics V2] The number of stream connections,0,kong,,
+kong.nginx.hconnections.total,gauge,,connection,,[OpenMetrics V2] (Kong V3) The number of HTTP and stream connections,0,kong,,
 kong.stream.status.count,count,,request,,[OpenMetrics V2] The stream status codes per service/route in Kong,0,kong,,
+kong.node.info,gauge,,,,[OpenMetrics V2] (Kong V3) Gauge set to 1 that outputs the nodes’s ID and Kong Gateway version,0,kong,,
```

Comment:
Validation is still failing for this (I think it's because of a strange quote on the description).

However, considering that the docs say:

> `node_info`: Single gauge set to 1 that outputs the node’s ID and Kong Gateway version.

This doesn't look useful as a metric, so I'd remove it altogether. It would be interesting to extract the version from it and either set it as metadata or as a tag for other metrics, but this would be more involved, so dropping it for now is fine.

Reply:
I @alopezz, I went ahead and removed that metric. Let me know if I can help test/add anything else.